### DISCUSSION TITLE
Add event.category field to firewall logs

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#8109] [Ideas] [answered]
Hello all,

Is it possible to add the event.category field to the filterlog parser?

So that in the SOC dashboards there is no "missing" behind firewall

![Knipsel](https://user-images.githubusercontent.com/29328171/173177260-78dda875-3cc5-44ae-a551-ce510997b62e.PNG)

Proposal : network or boundary

Regards

Bart

**Comment by dougburks** [comment, marked as answer]
I've created issue https://github.com/Security-Onion-Solutions/securityonion/issues/8112 for this.
